Formula/p/porter.rb

```diff
@@ -0,0 +1,34 @@
+class Porter < Formula
+  desc "App artifacts, tools, configs, and logic packaged as distributable installer"
+  homepage "https://porter.sh"
+  url "https://github.com/getporter/porter/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "eb5451b85e50e4033a50101171f8c6372ffd2600361bd34fa2d1166a8d062742"
+  license "Apache-2.0"
+  head "https://github.com/getporter/porter.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    cd "cmd/porter" do
+      ldflags = %W[
+        -s
+        -X get.porter.sh/porter/pkg.Version=#{version}
+        -X get.porter.sh/porter/pkg.Commit=#{tap.user}
+      ]
+
+      system "go", "build", *std_go_args(ldflags:)
+      generate_completions_from_executable(bin/"porter", "completion")
+      system bin/"porter", "mixin", "install", "exec", "--version", "v#{version}"
```

Comment:
is this installing a binary? we'll need to install it from source, is it just https://github.com/getporter/porter/tree/main/pkg/exec?

Reply:
> is this installing a binary? we'll need to install it from source, is it just https://github.com/getporter/porter/tree/main/pkg/exec?

Thank you for your review.

Based on the [document](https://porter.sh/docs/getting-started/install-porter/#mixins), we'll need to install it from the source.

Reply:
Also, it looks like this installs to `~/.porter`? Nothing outside the prefix will end up in the bottle, so we either can't do this or need to find a way for porter to find it within `brew --prefix porter`

Reply:
@branchvincent 

Thank you for the information. It appears that we cannot change the `mixin` installation directory in this version, so I have removed the `mixin install` command. We will update the brew installation documentation accordingly for users.

Reply:
sounds good